src/components/Person/Person.jsx

```diff
@@ -1 +1,33 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+      {age
+        ? (
+          <p className="Person__age">
+            {`I am ${age}`}
+          </p>
+        )
+        : null
+      }
```

Comment:
think of a way to avoid ternary in this case, there's a simpler way. You don't need to return null